### PR TITLE
Translate Audit pages

### PR DESCRIPTION
**Pull request by iant**
test coverage in `bundle exec rspec spec/system/schools/audits_spec.rb`

### CODE REVIEW

File: app/views/schools/audits/_form.html.erb [hunk 1]
```diff
@@ -60,31 +58,29 @@
...
-      <%= link_to_add_association 'Add activity', f, :audit_activity_types, class: 'btn' %>
+      <%= link_to_add_association t('schools.audits.form.add_activity'), f, :audit_activity_types, class: 'btn' %>
     </div>
   </div>
 
   <br/>
   <br/>
 
-  <h2>Actions</h2>
+  <h2><%= t('schools.audits.form.actions') %></h2>
```
Comment (urbanwide) [inline app/views/schools/audits/_form.html.erb:68]: Could this use common.labels ?

File: app/views/schools/audits/index.html.erb [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
       <table class="table" id="audits">
         <thead>
           <tr>
-            <th>Name</th>
-            <th>Completed on</th>
+            <th><%= t('schools.audits.index.name') %></th>
+            <th><%= t('schools.audits.index.completed_on') %></th>
             <% if can? :manage, @activity %>
-              <th>Actions</th>
+              <th><%= t('schools.audits.index.actions') %></th>
```
Comment (urbanwide) [inline app/views/schools/audits/index.html.erb:20]: and this? common.labels.actions ? Not sure if it's there but feels like it should be

File: app/views/schools/audits/show.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<% content_for :page_title, "#{@school.name} Audit - #{@audit.title}" %>
+<% content_for :page_title, "#{@school.name} #{t('schools.audits.show.title')} - #{@audit.title}" %>
```
Comment (urbanwide) [inline app/views/schools/audits/show.html.erb:1]: Should this whole line be translated, as the sentence order might be different in Welsh?

**Review by urbanwide — APPROVED** [review]
Looks good @iant - another few minor comments / queries 👍

File: app/views/schools/audits/_form.html.erb (hunk 1 above)
Comment (iant) [reply app/views/schools/audits/_form.html.erb]: :+1:

File: app/views/schools/audits/index.html.erb (hunk 2 above)
Comment (iant) [reply app/views/schools/audits/index.html.erb]: :+1:

File: app/views/schools/audits/show.html.erb (hunk 3 above)
Comment (iant) [reply app/views/schools/audits/show.html.erb]: :+1: have pushed updates to this and the above for actions.  would you be able to give this another quick review before i merge.  thanks :)

**Review by urbanwide — APPROVED** [review]
Looks good @iant 👍

**Pull request merged**